src/pages/index.tsx

```diff
@@ -4,6 +4,7 @@ import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
 import Layout from "@theme/Layout";
 import HomepageFeatures from "@site/src/components/HomepageFeatures";
 import Heading from "@theme/Heading";
+import LogoIcon from '@site/src/components/HomepageFeatures/Logo';
```

Comment:
I think the import path should be lowercase:

'@site/src/components/HomepageFeatures/logo'

> The 'l' in the 'logo'